oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -567,14 +567,15 @@ private APIGatewayProxyResponseEvent handleAuthJourney(
         var session = existingSession.orElseGet(sessionService::createSession);
         attachSessionIdToLogs(session);
 
+        Optional<String> previousSessionId = existingSession.map(Session::getSessionId);
         if (existingSession.isEmpty()) {
             updateAttachedSessionIdToLogs(session.getSessionId());
             LOG.info("Created session");
         } else {
-            var oldSessionId = session.getSessionId();
+            previousSessionId = Optional.of(session.getSessionId());
             sessionService.updateSessionId(session);
             updateAttachedSessionIdToLogs(session.getSessionId());
-            LOG.info("Updated session id from {} - new", oldSessionId);
+            LOG.info("Updated session id from {} - new", previousSessionId);
```

Comment:
Random question but what does passing an optional look like when logged? Just wondering if it actually logs out a meaningful value and not something weird

Reply:
Good question - it looks like `"Updated session id from Optional[5RdN6mlCmeLp2Q-3s6FbSdS3Anc] - new"`